src/components/Guideline.js

```diff
@@ -21,6 +21,12 @@ export function Guideline(props) {
         <ImageComponent>
           <img src={`./figma/${image}`} alt="" />
         </ImageComponent>
+        {goodImage && <Good className="mb-16" >
+          <img src={`./figma/${goodImage}`} alt="" />
+        </Good> }
+         {badImage && <Bad>
+          <img src={`./figma/${badImage}`} alt="" />
+        </Bad> }
```

Comment:
Here you would access the prop and do something like `props.images.good.map(..)` to map through and render the images within the appropriate container. Keep in mind that if you do this, you'd have to update this change for every existing instance of this component.